### PR TITLE
Adds wallet properties

### PR DESCRIPTION
**Pull request by NejcZdovc**

### CODE REVIEW

File: include/bat/ledger/wallet_info.h [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  std::string probi;
+  unsigned int expiryTime;
+};
+
+LEDGER_EXPORT struct WalletInfo {
+  WalletInfo();
+  ~WalletInfo();
+  WalletInfo(const WalletInfo& info);
+  std::string altcurrency_;
+  std::string probi_;
```
Comment (bridiver) [inline include/bat/ledger/wallet_info.h]: probi?

Comment (NejcZdovc) [reply include/bat/ledger/wallet_info.h]: it's the representation of balance but it's a big number with 18 places. Same thing as satoshi for BTC

File: src/bat/ledger/ledger.cc [hunk 2]
```diff
@@ -137,4 +137,16 @@ ledger::Ledger* Ledger::CreateInstance(LedgerClient* client) {
   return new bat_ledger::LedgerImpl(client);
 }
 
+WalletInfo::WalletInfo () {}
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply src/bat/ledger/ledger.cc]: done

File: src/bat_helper.cc [hunk 3]
```diff
@@ -778,10 +802,37 @@ namespace braveledger_bat_helper {
...
+        }
+      } else {
+        grants_.clear();
+      }
     }
     return !error;
   }
 
+  /////////////////////////////////////////////////////////////////////////////
+  GRANT::GRANT() {}
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply src/bat_helper.cc]: done

File: src/bat_helper.h [hunk 4]
```diff
@@ -149,6 +149,32 @@ namespace braveledger_bat_helper {
...
+    WALLET_PROPERTIES_ST();
+    ~WALLET_PROPERTIES_ST();
+    WALLET_PROPERTIES_ST(const WALLET_PROPERTIES_ST& properties);
+
+    //load from json string
+    bool loadFromJson(const std::string & json);
+
+    std::string altcurrency_;
+    std::string probi_;
+    double balance_;
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply src/bat_helper.h]: done

File: src/ledger_impl.cc [hunk 5]
```diff
@@ -298,4 +287,32 @@ void LedgerImpl::OnMediaRequestCallback(uint64_t duration, const braveledger_bat
   // SaveVisit(mediaPublisherInfo.publisher_, duration, true);
 }
 
+void LedgerImpl::OnWalletProperties(braveledger_bat_helper::WALLET_PROPERTIES_ST properties) {
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply src/ledger_impl.cc]: done

**Pull request merged**